### PR TITLE
Тайпинги

### PR DESCRIPTION
**Pull request by Ploffi**
Добавил файлы декларации, пофиксил тесты (jsdom localStorage)

### CODE REVIEW

File: src/array/any.ts [hunk 1]
```diff
@@ -23,4 +24,4 @@ export default curryN(2, (fn, arr = []) => {
     }
 
     return false;
-});
+}) as typeof ramdaAny
```
Comment (osorokotyaga) [inline src/array/any.ts]: please, remove ramda mentions, let it be just 'any'

File: src/array/each.ts [hunk 2]
```diff
@@ -15,7 +15,17 @@ import curryN from '../function/curryN';
...
-    for (let i = 0; i < arr.length; i++) {
-        fn(arr[i], i, arr);
-    }
-});
+  for (let i = 0; i < arr.length; i++) {
+    fn(arr[i], i, arr);
+  }
+}) as each
+
+interface each {
```
Comment (osorokotyaga) [inline src/array/each.ts]: should we move such definitons to typings/types to be more consistent?

File: src/array/reduce.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import * as R from '../typings/types';
```
Comment (osorokotyaga) [inline src/array/reduce.ts]: same here, remove ramda mentions

File: releaseUtils/build.js [hunk 4]
```diff
@@ -1,9 +1,13 @@
 /* eslint-disable import/no-commonjs, global-require */
 const execa = require('execa');
 
-const build = buildDir =>
+const build = (buildDir, tsTemp) =>
     execa.shell(
-        `npx babel --ignore __benchmarks__,__tests__ ./src -d ${buildDir}`
-    );
+        `tsc -p tsconfig.json`
```
Comment (markelog) [inline releaseUtils/build.js]: We should add a eslint config here, there is no need for backticks

Comment (Ploffi) [reply releaseUtils/build.js]: I think it's better to do this in a separate task

**Pull request merged**